### DISCUSSION TITLE
How do you run emulators on AWS EC2 instances

### DISCUSSION THREAD

**Discussion by prolificcoder** [MobileNativeFoundation/discussions#223] [General]
Hello,

I am wanting to give android emulators on EC2 a try. Can someone point out their configuration on EC2.

Details like
1. EC2 instance type
2. android/java version
3. avd type
4. Headless vs head (ful )
5. Linux machine vs Mac M2 machine

Even better if there is a script/steps I can use to boot up my own. 

**More context:**

I am trying to boot up emulators to run some UI tests, we are able to do that successfully for iOS on AWS mac but not for Android. 
We can try both Mac M1/M2 machines or linux runners.  

(No, I am not looking at device farms for this task) 

Appreciate pointers.

**Comment by esoxjem** [comment]
@prolificcoder afaik Android Emulators require virtualisation, so you might have to try a bare metal instance on EC2. Alternatively, try Genymotion on EC2
https://docs.genymotion.com/paas/02_Getting_Started/021_AWS/
